step1/README.md

```diff
@@ -47,24 +49,46 @@ This role will require the following permissions:
 
 Paste the following commands, replacing `{Subnet-Value}` and `{VPC-Value}`.  
 
-You can find these in the AWS console, by navigating to the `VPC` service, and selecting the ID of a public subnet, or one that allows port 80 inbound.  
+You can find these in the AWS console, [by navigating to the VPC service, and selecting the ID of a public subnet.](https://ap-southeast-2.console.aws.amazon.com/vpc/home?region=ap-southeast-2)
+
+*Note:* Subnet is considered public if it has direct access to internet via internet gateway. If you
+are operating within default VPC created by AWS on your behalf, during account creation, all subnets
+will be public by default. VPC ID can also be discovered from subnet's console, in "VPC" column. 
+
+<img alt="public subnet img" src="https://user-images.githubusercontent.com/1170273/142021113-b45706a5-6585-477e-9e70-68ef4b89719b.png" />
+
 
 ``` aws
-aws cloudformation create-stack --stack-name ec2-metadata-ssrf --template-body file://vuln-ec2-template.yaml --parameters ParameterKey="WebServerIngressLocation",ParameterValue="$IP_ADDRESS/32" ParameterKey="Subnet",ParameterValue="{Subnet-Value}" ParameterKey="DeploymentVPC",ParameterValue="{VPC-Value}" --capabilities CAPABILITY_NAMED_IAM
+aws cloudformation create-stack --stack-name ec2-metadata-ssrf \
+    --template-body file://vuln-ec2-template.yaml \
+    --parameters \
+     ParameterKey="WebServerIngressLocation",ParameterValue="$IP_ADDRESS/32" \
+     ParameterKey="Subnet",ParameterValue="{Subnet-Value}" \
+     ParameterKey="DeploymentVPC",ParameterValue="{VPC-Value}" \
+     --capabilities CAPABILITY_NAMED_IAM --region eu-south-1
 ```
 
-Open up cloudformation in the console and validate that the deployment has succeeded.
+
+
+Open up [cloudformation in the console](https://ap-southeast-2.console.aws.amazon.com/cloudformation/home?region=ap-southeast-2#/stacks?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false) and validate that the deployment has succeeded.
 
 <br />
 <br />
-4.  Fill the bucket
 
-Find the name of the bucket, and upload the text file to the recently created bucket.  This will serve as our mock customer information. 
+#### 4.  Fill the bucket
+
+
+CloudFormation stack named `ec2-metadata-ssrf` deployed previously has creaated S3 Object Store bucket. 
+In this step, we will disvoer the name of tis bucket, and upload the text file to the recently created bucket. This will serve to demonstrate web application access to potentially sensitive and confidential 
```

Comment:
Could you change this to:

The CloudFormation stack named `ec2-metadata-ssrf` we deployed previously has created an S3 Object Store bucket. 
In this step, we will discover the name of this bucket, and upload the text file to the recently created bucket. This will serve to demonstrate web application access to potentially sensitive and confidential information